apps/src/templates/teacherDashboard/TeacherDashboardHeader.jsx

```diff
@@ -88,32 +91,43 @@ function TeacherDashboardHeader({
   };
 
   return (
-    <div style={styles.headerContainer}>
-      <SmallChevronLink
-        href="/home#classroom-sections"
-        text={i18n.viewAllSections()}
-        iconBefore
-        style={styles.linkPadding}
-      />
+    <div className={dashboardStyles.headerContainer}>
```

Comment:
Looks good!  I am wondering if `SmallCheveronLink` will be updated by Denys at some point, OR if we should update it now?  This looks fantastic but I am wondering this could be it's own reusable component (ex. `SmallCheveronLinkBrandRefresh`) ???

Reply:
The `SmallChevronLink` was only used in one other place, and it's really a specialized version of a more general case where you want a link with an icon next to it. The problem with putting the icon inside of the `Link` was that our new link styling puts an underline under the icon when you do that. I think support for non-underlined icons next to links in our standard components would be a useful feature to add, but the best way to do that in a standardizable way would probably require some careful thought and I feel like it's beyond the scope of this task.

Reply:
Makes sense!  I didn't realize the new styling would conflict here - thanks for that context!